spawn/cfg.go

```diff
@@ -166,7 +166,7 @@ func (cfg *NewChainConfig) CreateNewChain() error {
 	// Set proper pairings for modules to be disabled if others are enabled
 	cfg.SetProperFeaturePairs()
 
-	logger.Debug("Spawning new app", "app", NewDirName)
+	logger.Info("Spawning new app", "name", NewDirName)
```

Comment:
my Windows dell machine is very slow, and I did not see an output. So thought it was broken. nice to have for older hardware